mmv1/third_party/terraform/services/securitycentermanagement/resource_scc_management_organization_event_threat_detection_custom_module_test.go

```diff
@@ -14,8 +14,26 @@ import (
 	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
 )
 
+func TestAccSecurityCenterManagement(t *testing.T) {
+	testCases := map[string]func(t *testing.T){
+		"organization": TestAccSecurityCenterManagementOrganizationEventThreatDetectionCustomModule,
+	}
+
+	for name, tc := range testCases {
+		// shadow the tc variable into scope so that when
+		// the loop continues, if t.Run hasn't executed tc(t)
+		// yet, we don't have a race condition
+		// see https://github.com/golang/go/wiki/CommonMistakes#using-goroutines-on-loop-iterator-variables
+		tc := tc
+		t.Run(name, func(t *testing.T) {
+			tc(t)
+		})
+	}
+}
+
+
 func TestAccSecurityCenterManagementOrganizationEventThreatDetectionCustomModule(t *testing.T) {
```

Comment:
```suggestion
func testAccSecurityCenterManagementOrganizationEventThreatDetectionCustomModule(t *testing.T) {
```

Reply:
Changing the test starting with lowercase letter to avoid it being run separately. Instead, it will be run in the test `TestAccSecurityCenterManagement`.

Reply:
The same change is needed for test `TestAccSecurityCenterManagementOrganizationSecurityHealthAnalyticsCustomModule`

Reply:
addressed